src/lib/TranscriptEditor/index.js

```diff
@@ -52,43 +62,51 @@ class TranscriptEditor extends React.Component {
   componentDidUpdate(prevProps, prevState) {
     // Transcript and media passed to component at same time
     if (
-      (prevState.transcriptData !== this.state.transcriptData)
-        && (prevProps.mediaUrl !== this.props.mediaUrl )
+      prevState.transcriptData !== this.state.transcriptData &&
+      prevProps.mediaUrl !== this.props.mediaUrl
     ) {
       console.info('Transcript and media');
       this.ifPresentRetrieveTranscriptFromLocalStorage();
     }
     // Transcript first and then media passed to component
     else if (
-      (prevState.transcriptData === this.state.transcriptData)
-      && (prevProps.mediaUrl !== this.props.mediaUrl)
+      prevState.transcriptData === this.state.transcriptData &&
+      prevProps.mediaUrl !== this.props.mediaUrl
     ) {
       console.info('Transcript first and then media');
       this.ifPresentRetrieveTranscriptFromLocalStorage();
     }
     // Media first and then transcript passed to component
     else if (
-      (prevState.transcriptData !== this.state.transcriptData)
-      && (prevProps.mediaUrl === this.props.mediaUrl)
+      prevState.transcriptData !== this.state.transcriptData &&
+      prevProps.mediaUrl === this.props.mediaUrl
     ) {
       console.info('Media first and then transcript');
       this.ifPresentRetrieveTranscriptFromLocalStorage();
     }
   }
 
   ifPresentRetrieveTranscriptFromLocalStorage = () => {
```

Comment:
is there a better name for this..?

Reply:
for `ifPresentRetrieveTranscriptFromLocalStorage`?